pkg/store/cache/inmemory_test.go

```diff
@@ -45,19 +45,14 @@ max_item_size: 2KB
 	testutil.Equals(t, uint64(1024*1024), cache.maxSizeBytes)
 	testutil.Equals(t, uint64(2*1024), cache.maxItemSizeBytes)
 
-	// Should instance an in-memory index cache with specified YAML config.s with units.
+	// Should return error when max_size is lesser than max_item_size of cache.
```

Comment:
Grammar nit: `less` is used for comparing whereas `lesser` is an adjective.

```suggestion
	// Should return error when max_size is less than max_item_size of cache.
```